modules/negotiate.go

```diff
@@ -171,6 +181,14 @@ var (
 		Standard: uint64(1 << 22), // 4 MiB
 		Testing:  uint64(1 << 12), // 4 KiB
 	}).(uint64)
+
+	// NegotiateMetadataMaxSliceSize establishes the maximum allowed length
+	// of the list of sectors returned by the Metadata RPC.
+	NegotiateMetadataMaxSliceSize = build.Select(build.Var{
+		Dev:      uint64(1 << 17),
+		Standard: uint64(1 << 17),
+		Testing:  uint64(1 << 4),
```

Comment:
these numbers are kind of magic; better would be `modules.SectorSize / crypto.HashSize`.